### PR TITLE
chore: export type definitions react components

### PR DESCRIPTION
**Pull request by 9renpoto**
closes https://github.com/tailwindlabs/heroicons/issues/64

### CODE REVIEW

File: scripts/build-react.js [hunk 1]
```diff
@@ -1,21 +1,22 @@
 const fs = require('fs').promises
-const dedent = require('dedent')
```
Comment (9renpoto) [inline scripts/build-react.js:2]: unused

File: scripts/build-react.js [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
 const rimraf = promisify(require('rimraf'))
 const svgr = require('@svgr/core').default
 
+const rootDir = './react'
+
 console.log(svgr)
 
 function svgToReact(svg, componentName) {
-  return svgr(svg, {}, { componentName })
+  return svgr(svg, { typescript: true }, { componentName })
```
Comment (9renpoto) [inline scripts/build-react.js:12]: https://github.com/gregberge/svgr/blob/b5920550bd966f876cb65c5e23af180461e5aa23/packages/core/src/config.js#L8

**Comment by pspeter3** [comment]
Is there still appetite for shipping this change?

**Comment by bradlc** [comment]
Hey @9renpoto. Sorry for missing this when you first opened it! We have since added type declarations for the components so I'm going to close this. I appreciate you taking the time to put this together though!

**Pull request closed without merging**